lib/assets/javascripts/cdb

```diff
@@ -1 +1 @@
-Subproject commit a2b8834622f2b4d3b849a7e810cff998d982844d
+Subproject commit b308a025d124af7b11144a95adf9bca8e47b7176
```

Comment:
Looks like this got accidentally updated somehow. If you didn't make any changes to cartodb.js then the removed `a2bb83` commit is the correct one.